### ISSUE TITLE
Check label elements for inconsistencies and correct associations

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1668]
Relevant Components
- [ ] Document Information
- [ ] File Upload

Description
The labeling of the inputs on this page is inconsistent. There is a <label> element that is not associated with the input, there is a name attribute that matches the label, and there is reminder text saying something different. With all of these, the information will be repeated, which is confusing for users.

Use ```<label>``` with an attribute “for” listing the id of the associated input.

Current Code
```
<label>
  text
</label>
...
<input>
```

Solution
Make sure labels are programmatically associated with the input for which they are intended to provide information.
Recommended Code
```
<label for="id">
  text
</label>
...
<input id="id">
```

**Comment by jcreel** [comment]
Deliverable for spike:  Survey, determine feasibility, decompose, and size task of providing ids for inputs and remediating all the label associations.

**Comment by jrmartino** [comment]
We have discovered this same issue during an accessibility audit, and are nearly done with the fixes at JHU.

**Comment by jrmartino** [comment]
Sorry for the lapse - my colleague Jared has just placed a PR on our local fork which addresses this issue and also the issue of jumping header levels. I am looking into this. There are also some commits which have to do with our JHU branding, so I need to look and see how tangled these concerns are. 

His changes are in the jarred-ui-tweaks branch of https://github.com/jhu-sheridan-libraries/Vireo

I am looking to review this early next week.
